rust/openvasd/src/storage/mod.rs

```diff
@@ -186,6 +186,11 @@ pub trait NVTStorer {
 
     /// Returns the currently stored feed hash.
     async fn feed_hash(&self) -> Vec<FeedHash>;
+
+    /// Returns the current feed version, if any.
+    async fn current_feed_version(&self) -> Result<String, Error> {
+        Ok("".to_string())
+    }
```

Comment:
Just a question about this: Do the other storage types not have any information about the feed version?

Reply:
It seems that the metadata is completely replace in the other storage types, both `memory` and `fs` when the daemon is restarted.